docs/lib/datastore/data-access.md

```diff
@@ -64,6 +64,18 @@ Alternatively, the `or` logical operator can also be used:
 <inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/data-access/query-predicate-or-snippet.md"></inline-fragment>
 <inline-fragment platform="android" src="~/lib/datastore/fragments/android/data-access/query-predicate-or-snippet.md"></inline-fragment>
 
+### Sort
+
+Results can be queried by order you specify, either sorted by one field or multiple fields.
+
+For one field sorting example, sort by `rating` in ascending order:
+
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/data-access/query-sort-snippet.md"></inline-fragment>
```

Comment:
from Android docs, this will look very odd because it is missing the fragment. 
- Does Android have sorting capability? if so, can you add Android fragment as well? and we can get Android folks to help out on the code snippet
- If not, is there some Issue to track this feature on Android? if not, i think the fragment should contain the description and heading as well so that it's completely hidden from Android docs

Reply:
Let me check with Richard M.